### PR TITLE
Mkrasne2/og3 82

### PR DESCRIPTION
**Pull request by mzkrasner**
Additional updates to tests page, and adding in Consumerfactory (we may use it on test page)

### CODE REVIEW

File: pages/api/kite/connect/kafka.ts [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import type { NextApiRequest, NextApiResponse } from 'next/types';
 import Kite from '@/common/kite';
 import ProducerFactory from '@/common/kafkaConnector/ProducerFactory';
+import ConsumerFactory from '@/common/kafkaConnector/ConsumerFactory';
```
Comment (zahara-aviv) [inline pages/api/kite/connect/kafka.ts:4]: Why is this library imported if its never used?

File: pages/api/kite/connect/listTopics.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import ConsumerFactory from '@/common/kafkaConnector/ConsumerFactory';
+
+type Data = {
+    topics?: string[];
+    err?: unknown;
+  };
+// eslint-disable-next-line import/no-anonymous-default-export 
+export default async (req: NextApiRequest, res: NextApiResponse<Data>) => {
+
+  const kafkaSetup = Kite.getKafkaSetup();
```
Comment (zahara-aviv) [inline pages/api/kite/connect/listTopics.ts:14]: I think its best to await the getKafkaSetup in case its remote.

File: pages/api/socket.ts [hunk 3]
```diff
@@ -2,11 +2,8 @@ import { NextApiRequest } from "next/types";
 import { NextApiResponseServerIO } from "src/common/types/io";
 import { Server as ServerIO } from "socket.io";
 import { Server as NetServer } from "http";
-import ExampleConsumer from '../../src/common/utilities/consumerUtil';
-
-const clientId = 'json1';
-const brokers = ['localhost:9092', 'localhost:9093'];
-const topic = 'humanTopic';
+import Kite from '@/common/kite';
+import ConsumerFactory from '@/common/kafkaConnector/ConsumerFactory';
```
Comment (zahara-aviv) [inline pages/api/socket.ts:6]: imported but not used

File: pages/api/kite/connect/kafka.ts (hunk 1 above)
Comment (mzkrasner) [reply pages/api/kite/connect/kafka.ts]: Commented out for now (with comments) but left in the file for now -> was thinking we might want to return data from the consumer to the frontend regarding what partition a new message was assigned to, etc. Final version of tests page will determine usage, at which point we can remove or uncomment

File: pages/api/socket.ts (hunk 3 above)
Comment (mzkrasner) [reply pages/api/socket.ts]: Commented out, similar thought process as above, but more detailed comments left next to the commented out imports

**Review by zahara-aviv — APPROVED** [review]

**Pull request merged**